.github/workflows/mobile-e2e.yml

```
@@ -0,0 +1,195 @@
+            os: macos-latest
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - run: corepack enable
+      - run: corepack prepare yarn@4.6.0 --activate
+      - run: yarn install --immutable --silent
+      - name: Install Maestro
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:30: Double quote to prevent globbing and word splitting

(shellcheck)